data/map.txt

```diff
@@ -27973,10 +27969,9 @@ planet Palatine
 	government Uninhabited
 
 planet Parii'het
-	attributes ka'het
+	attributes ka'het uninhabited
 	landscape land/space6
-	description `Parii'het was almost finished when the invasion broke out, and already capable of (slowly) creating Ka'het. Thanks to the mining operations on Alix and the occasional harvest taken here by the youngest Ka'het, most of the facilities are still fully operational even to this day.`
-	description `	Differently from most of the Graveyard, the station doesn't seem to have ever suffered any damage from the Ka'het, nor from the Builders trying to regain control over it.`
+	description `Very similar to its sister station of Seli'het, Parii'het was still under construction when it was abandoned by its creators. Today it floats uninhabited in this rarely visited section of space, approached occasionally by the automated drones that mindlessly keep carrying new supplies to the massive cargo section. If the Ka'het attacked this station, it would be made entirely inoperable in a short time.`
```

Comment:
```suggestion
	description `Very similar to its sister station of Seli'het, Parii'het was still under construction when it was abandoned by its creators. Today, it floats uninhabited in this rarely visited section of space, approached occasionally by the automated drones that mindlessly continue to carry new supplies to the massive cargo section. If the Ka'het attacked this station, it would be made entirely inoperable in a short time.`
```